tree.go

```diff
@@ -1467,7 +1467,7 @@ func (n *LeafNode) GetProofItems(keys keylist, _ NodeResolverFn) (*ProofElements
 // Serialize serializes a LeafNode.
 // The format is: <nodeType><stem><bitlist><comm><c1comm><c2comm><children...>
 func (n *LeafNode) Serialize() ([]byte, error) {
-	cBytes := banderwagon.ElementsToBytesUncompressed([]*banderwagon.Element{n.commitment, n.c1, n.c2})
+	cBytes := banderwagon.BatchToBytesUncompressed(n.commitment, n.c1, n.c2)
```

Comment:
Be coherent with any method that does "batching" of work.
Before we had:
- `MultiToMapFieldElement(...)`, and
- `ElementsToBytesUncompressed(...)`.

Now I switched to a more coherent naming:
- `BatchToMapFieldElement(...)`, and
- `BatchToBytesUncompressed(...)`.

Also made the latter variadic args.